static/panes/executor.js

```diff
@@ -177,7 +177,7 @@ Executor.prototype.resize = function () {
 };
 
 function errorResult(message) {
-    return {code: -1, stderr: message};
+    return {code: -1, stdout: [], stderr: [message]};
```

Comment:
should this probably not be `stderr: [{text: message}]` or is it turned into something else at this point?

Reply:
Oh, there's a bug with the TS conversion here, you're totally right, I'll try to fix it

Reply:
Fixed now. Moved some types around too. At some point we need to review our API and check that the returns are actually what the types say they are